addons/calendar/models/calendar_event.py

```diff
@@ -263,7 +263,11 @@ def _compute_current_attendee(self):
             event.current_attendee = current_attendee and current_attendee[0]
 
     def _search_current_attendee(self, operator, value):
-        return [("id", operator, value)]
+        attendees = self.env['calendar.attendee'].search([
+            ('partner_id', '=', self.env.user.partner_id.id),
+            ('id', operator, value)
+        ])
+        return [('attendee_ids', 'in', attendees.ids)]
```

Comment:
```suggestion
        return [
            ('attendee_ids', 'any', [
                ('partner_id', '=', self.env.user.partner_id.id),
                ('id', operator, value)
            ])
        ]
```
this could avoid doing an extra query if it works